common/src/main/java/org/apache/uniffle/common/config/RssConf.java

```diff
@@ -569,7 +573,7 @@ public <T> RssConf set(ConfigOption<T> option, T value) {
     return this;
   }
 
-  <T> void setValueInternal(String key, T value) {
+  public <T> void setValueInternal(String key, T value) {
```

Comment:
This method should be used in the class inner from the name. Could we avoid using it?

Reply:
I don't use this, let me recover.